src/components/common/item-selector/item-selector.tsx

```diff
@@ -178,16 +192,16 @@ class ItemSelector extends Component<ItemSelectorProps> {
     // only used when multiSelect = true
     e.preventDefault();
     e.stopPropagation();
-    const {selectedItems} = this.props;
```

Comment:
@heshan0131

due to ts cannot understand that selectedItems should be array in this case. Btw, in some cases it can be wrong - we can use multiSelect = true, but selectedItems not array type